Solutions/NXLogAixAudit/Data Connectors/NXLogAixAudit.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "NXLogAixAudit",
+    "title": "NXLog AIX Audit",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog [AIX Audit](https://nxlog.co/documentation/nxlog-user-guide/im_aixaudit.html) data connector uses the AIX Audit subsystem to read events directly from the kernel for capturing audit events on the AIX platform. This REST API connector can efficiently export AIX Audit events to Azure Sentinel in real time.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**NXLog_parsed_AIX_Audit_view**](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/NXLogAixAudit/Parsers/NXLogAixAudit.txt) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "",
+            "baseQuery": "NXLog_parsed_AIX_Audit_view"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "AIX Audit event type distributon",
+            "query": "NXLog_parsed_AIX_Audit_view\n| summarize count() by EventType\n| render piechart title=\"AIX Audit event type distributon\""
+        },
+        {
+            "description" : "Highest Event per Second (EPS) AIX Audit vent types",
+            "query" : "NXLog_parsed_AIX_Audit_view\n| where EventTime >= todatetime('2021-08-16')\n| where EventTime <  todatetime('2021-08-20')\n| summarize Count=count() by EpsDateTime, EventType\n| summarize EPS=max(Count) by EventType\n| sort by EPS\n| take 5\n| render columnchart title=\"Highest Event per Second (EPS) Event Types\""
+        },
+        {
+            "description" : "Time chart of AIX Audit events per day",
+            "query": "NXLog_parsed_AIX_Audit_view\n| where EventTime >= todatetime('2021-08-16')\n| where EventTime <  todatetime('2021-08-20')\n| summarize Count=count() by bin(EventTime, 1d)\n| render timechart title=\"AIX Audit events per day\""
+        },
+        {
+            "description" : "Time chart of AIX Audit events per hour",
+            "query": "NXLog_parsed_AIX_Audit_view\n| where EventTime >= todatetime('2021-08-17')\n| where EventTime <  todatetime('2021-08-18')\n| summarize Count=count() by bin(EventTime, 1h)\n| render timechart title=\"AIX Audit events per hour\""
+        },
+        {
+            "description" : "AIX Audit Events per Second (EPS) Time Chart",
+            "query": "NXLog_parsed_AIX_Audit_view\n| where EventTime >= todatetime('2021-08-17 18:29')\n| where EventTime <  todatetime('2021-08-17 23:55')\n| summarize EPS=count() by bin(EventTime, 1s)\n| render timechart title=\"AIX Audit Events per Second (EPS)\""
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "",

```

Comment:
Name should be here as data type name. Which is **NXLog_parsed_AIX_Audit_view**

Reply:
Applied your suggestion in commit [c9deb14e]( https://github.com/Azure/Azure-Sentinel/pull/2968/commits/c9deb14e8010dcba35f2da02a188ea6dea182e0a)